frontend/src/actionCreators/accessRequestActionCreator.tsx

```diff
@@ -0,0 +1,129 @@
+import { showLoading, hideLoading } from 'react-redux-loading-bar';
+import { request, success, error } from 'actions/genericActions';
+import * as reducerTypes from 'constants/reducerTypes';
+import * as API from 'constants/API';
+import {
+  IStoreAccessRequestsAction,
+  storeAccessRequests,
+  storeAccessRequest,
+} from 'actions/accessRequestActions';
+import { IAccessRequest } from 'interfaces/accessRequests';
+import * as actionTypes from 'constants/actionTypes';
+import { ENVIRONMENT } from 'constants/environment';
+import CustomAxios from 'customAxios';
+import { AxiosResponse, AxiosError } from 'axios';
+import { createRequestHeader } from 'utils/RequestHeaders';
+
+/**
+ * Converts the 'values' object into an IAccessRequest object
```

Comment:
Can we start using `tsdoc` everywhere?  I find it very difficult to navigate through our current code without it.